samples/linkedin/README.md

```diff
@@ -0,0 +1,601 @@
+# Linkedin
+
+Sequence that reads CSV input (e.g. dump of companies from crunchbase), uses RapidAPI to gather additional data from LinkedIn and outputs aggregated details.
+
+## Pre-configuration
+
+Create `config.json` file with following content:
+
+```json
+{
+    "XRapidAPIKey": "YOUR_RAPID_API_KEY_GOES_HERE"
```

Comment:
@MichalCz I guess this is the "_problem_" - It's not `XRapidAPIKey` but `RAPID_API_KEY` - I'll update README.

Reply:
Done